infrastructure/gitsync-core/codepipeline/non-production/codestar-connection.yaml

```diff
@@ -0,0 +1,43 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  tagEnvironment:
+    Type: String
+    Default: "non-production"
+  tagOwner:
+    Type: String
+    Default: "fog"
+  tagService:
+    Type: String
+    Default: "aws-metric-streams-client"
+  tagSystem:
+    Type: String
+    Default: "fog"
+  tagProduct:
+    Type: String
+    Default: "GOV.UK One Login"
+
+Resources:
+  CodeStarConnection:
+    Type: 'AWS::CodeStarConnections::Connection'
+    Properties:
+      ConnectionName: GDS-GitHub-Connection
```

Comment:
This name will conflict with the LZA provisioned codeconnection if it's deployed to the same account - should it have a distinct name ?